normandy/recipes/models.py

```diff
@@ -129,7 +129,7 @@ def __str__(self):
         return self.name
 
     def save(self, *args, **kwargs):
-        if self.revision_id:
+        if self.revision_id is not None:
```

Comment:
When would the `revision_id` be `None`, anyways? The default is 0, and the database doesn't allow nulls, right?